packages/ssz/src/util/proof/treePartialToJsonPaths.ts

```diff
@@ -41,75 +41,67 @@ export function treePartialToJsonPaths(
       const childType = type.getPropertyType(jsonPathProp);
 
       // If this type merkleized fits in a single chunk then this LeafNode includes all data
-      if (childType.maxChunkCount === 1) {
-        return {type: TreeDataTypeCode.complete, jsonPathProps: [jsonPathProp]};
-      } else {
-        return {type: TreeDataTypeCode.witness};
-      }
+      if (childType.maxChunkCount === 1) return {type: TreeDataTypeCode.complete, jsonPathProps: [jsonPathProp]};
```

Comment:
Not having the brackets around conditional blocks is a recipe for bugs.  Even if there is only a single line I think we should keep them.  Only in some very rare cases where there are a bunch of very very short conditionals in sequence does this make sense


```ts
if (cond1) return "ans1";
if (cond2) return "ans2";
if (cond3) return "ans3";
if (cond4) return "ans4";
return "ans5";
```